sifive/platform.h

```diff
@@ -30,12 +30,22 @@
  *****************************************************************************/
 
 // CPU info
+#ifndef FPGA
```

Comment:
We should probably move the entire platform.h into platform specific versions, rather than have layers of IFDEFs in it.

Reply:
So what you mean is we should create a completely new **platform_fpga.h**, move the old content to **platform_hifive.h**, then from the **platform.h** we have just one **#ifndef FPGA** to select which one of them?

Reply:
Yes, I mean that we should have a `platform_fpga.h` and a `platform_hifive.h`, and then have `platform.h` use one of them based on the flag.

Makes it easier for us to port further.

Reply:
Alright, we just did that and tested it again.
Everything is working just fine.
The new commit is up : #1 db7a2cb
Thank you.